app/views/partners/requests/_item_request.html.erb

```diff
@@ -1,13 +1,25 @@
 <%= form.fields_for :item_requests, defined?(object) ? object : nil  do |field| %>
-  <tr>
+  <tr data-controller="item-units" data-item-units-item-units-value="<%= item_units.to_json %>">
     <td>
       <%= field.label :item_id, "Item Requested", {class: 'sr-only'} %>
-      <%= field.select :item_id, @requestable_items, {include_blank: 'Select an item'}, {class: 'form-control'} %>
+      <%= field.select :item_id, @requestable_items, {include_blank: 'Select an item'},
```

Comment:
So here we already only show the `@requestable_items` in the item select drop-down, so that already narrows down what we might show in the units drop down. So the generated `@item_units` having all possible items it works fine, but could be narrowed down to building off of `@requestable_items`